connector.go

```diff
@@ -169,6 +171,87 @@ func (connector *Connector) registerRoutes() {
 		fmt.Fprint(w, "Hi, octyne is online and listening to this port successfully!")
 	})
 
+	// POST /accounts
+	// PATCH /accounts
+	// DELETE /accounts
+	type accountsRequestBody struct {
+		Username string `json:"username"`
+		Password string `json:"password"`
+	}
+	connector.Router.HandleFunc("/accounts", func(w http.ResponseWriter, r *http.Request) {
+		if !connector.Validate(w, r) {
+			return
+		}
+		if r.Method == "GET" {
+			http.Error(w, "{\"error\":\"Method not allowed!!\"}", http.StatusMethodNotAllowed)
+			return
+		}
+		var users map[string]string
+		contents, err := ioutil.ReadFile("users.json")
+		if err != nil {
+			log.Println("Error reading users.json when modifying accounts!", err)
+			http.Error(w, "{\"error\":\"Internal Server Error!\"}", http.StatusInternalServerError)
+			return
+		}
+		err = json.Unmarshal(contents, &users)
+		if err != nil {
+			log.Println("Error parsing users.json when modifying accounts!", err)
+			http.Error(w, "{\"error\":\"Internal Server Error!\"}", http.StatusInternalServerError)
+			return
+		}
+		if r.Method == "POST" || r.Method == "PATCH" {
+			var buffer bytes.Buffer
+			_, err := buffer.ReadFrom(r.Body)
+			if err != nil {
+				http.Error(w, "{\"error\":\"Failed to read body!\"}", http.StatusBadRequest)
+				return
+			}
+			var body accountsRequestBody
+			err = json.Unmarshal(buffer.Bytes(), &body)
+			if err != nil {
+				http.Error(w, "{\"error\":\"Invalid JSON body!\"}", http.StatusBadRequest)
+				return
+			}
+			if body.Username == "" || body.Password == "" {
+				http.Error(w, "{\"error\":\"Username or password not provided!\"}", http.StatusBadRequest)
+				return
+			}
+			if r.Method == "POST" && users[body.Username] != "" {
+				http.Error(w, "{\"error\":\"User already exists!\"}", http.StatusConflict)
+				return
+			} else if r.Method == "PATCH" && users[body.Username] == "" {
+				http.Error(w, "{\"error\":\"User does not exist!\"}", http.StatusConflict)
+				return
+			}
+			sha256sum := fmt.Sprintf("%x", sha256.Sum256([]byte(body.Password)))
+			users[body.Username] = sha256sum
+		} else if r.Method == "DELETE" {
+			var username = r.URL.Query().Get("username")
+			if username == "" {
+				http.Error(w, "{\"error\":\"Username not provided!\"}", http.StatusBadRequest)
+				return
+			}
+			if users[username] == "" {
+				http.Error(w, "{\"error\":\"User does not exist!\"}", http.StatusConflict)
+				return
+			}
+			delete(users, username)
+		}
+		account, err := json.MarshalIndent(users, "", " ")
```

Comment:
Use 2-space indents.
Also newline at file end is good, apparently MarshalIndent doesn't do this (tested in Go playground in docs page)
Also here's a better variable name
```suggestion
		usersJson, err := json.MarshalIndent(users, "", "  ") + "\n"
```